Xamarin.Forms.Core/ListView.cs

```diff
@@ -421,7 +421,7 @@ public string GetDisplayTextFromGroup(object cell)
 		}
 
 		[EditorBrowsable(EditorBrowsableState.Never)]
-		public void NotifyRowTapped(int groupIndex, int inGroupIndex, Cell cell = null)
+		public void NotifyRowTapped(int groupIndex, int inGroupIndex, Cell cell = null, bool isContextMenuRequested = false)
```

Comment:
I think this is a breaking change. We need to make another overload that keeps the same signature. Right, @PureWeen ?

Reply:
@samhouts, am I understand you correct, that we'll have 4 methods?
void NotifyRowTapped(int index, int inGroupIndex, Cell cell, bool isContextMenuRequested); 
void NotifyRowTapped(int index, Cell cell, bool isContextMenuRequested); 
void NotifyRowTapped(int index, Cell cell);  
void NotifyRowTapped(int index, int inGroupIndex, Cell cell); 

For now code is not changed for execution if you not set isContextMenuRequested.

Reply:
@samhouts right, this breaks the ABI